backend/src/main/java/com/votogether/domain/auth/dto/response/OAuthDisconnectResponse.java

```diff
@@ -0,0 +1,6 @@
+package com.votogether.domain.auth.dto.response;
+
+public record OAuthDisconnectResponse(
+        Long id
+) {
+}
```

Comment:
```suggestion
public record OAuthDisconnectResponse(Long id) {
}
```
매개변수가 1개일 때도 내리는 편이신가요? 😀

Reply:
record는 하나일 때도 내려서 사용하는 것이 더 가독성이 좋은 것 같더라구요.
아마 기존의 클래스의 형태가 익숙해서 그런 것 같아요.